### PR TITLE
feat: add gitignore

### PR DESCRIPTION
**Pull request by matmarzolagomes**

### CODE REVIEW

File: scripts/test [hunk 1]
```diff
@@ -15,7 +15,6 @@ git config user.name "Test"
 ./gootstrap -group testgroup -name testservice
 
 echo "generated files on: ${tmpdir}"
-tree
```
Comment (guilherme-santos) [inline scripts/test:18]: Looks nice. Why remove it?

**Review by guilherme-santos — APPROVED** [review]

File: scripts/test (hunk 1 above)
Comment (matmarzolagomes) [reply scripts/test:18]: We thought that not everybody would have this installed (but that was my cas :rofl: ) maybe we can put it back? @katcipis

Comment (guilherme-santos) [reply scripts/test:18]: Ah yeah, it's a point as shell.. I said what I mean, looks nice but not a requirement. 

Maybe `command tree && tree` would do the trick? But you can ignore if you want

Comment (katcipis) [reply scripts/test:18]: yeah maybe, but one can always go to the dir the test created and run tree to investigate (or investigate some other way).

**Pull request merged**